.gitignore

```diff
@@ -23,3 +23,6 @@ testbin/*
 *.swp
 *.swo
 *~
+
+# Manually generated crds. Directory is needed though
+generated-crds/*
```

Comment:
The folder is needed so kustomize saves them in several files:
```
kubewarden-controller (no-cainjection-crds *$%)$ tree generated-crds
generated-crds
├── apiextensions.k8s.io_v1_customresourcedefinition_admissionpolicies.policies.kubewarden.io.yaml
├── apiextensions.k8s.io_v1_customresourcedefinition_clusteradmissionpolicies.policies.kubewarden.io.yaml
└── apiextensions.k8s.io_v1_customresourcedefinition_policyservers.policies.kubewarden.io.yaml
```

I chose not to check-in the files, but we could them check-in too.

Reply:
I think it's fine to not check them in.